flink-connector-mysql-cdc/src/test/java/com/ververica/cdc/connectors/mysql/table/MySqlConnectorITCase.java

```diff
@@ -1423,6 +1426,191 @@ public void testColumnOptionalWithDefaultValue() throws Exception {
         result.getJobClient().get().cancel().get();
     }
 
+    @Test
+    public void testUcs2Charset() throws Exception {
```

Comment:
I think we can move the charset tests in a single file. I think it would be much clean.

Reply:
OK, I will fix it.